packages/blocks/src/api/test/validation.js

```diff
@@ -417,7 +417,7 @@ describe( 'validation', () => {
 
 		it( 'should return true for effectively equivalent html', () => {
 			const isEquivalent = isEquivalentHTML(
-				'<div>&quot; Hello<span   class="b a" id="foo" data-foo="here &mdash; there"> World! &#128517;</  span>  "</div>',
```

Comment:
We should probably have some test cases for `isEqualTagAttributePairs` and/or `isEqualTokensOfType` which explicit test the case-insensitive behavior. These specific test cases are admittedly more of a "catch-all" hodgepodge.

Reply:
Done in 255aee0